src/vs/code/electron-main/app.ts

```diff
@@ -934,8 +936,7 @@ export class CodeApplication extends Disposable {
 				cli: { ...this.environmentMainService.args },
 				forceNewWindow: true,
 				forceEmpty: true,
-				gotoLineMode: true,
-				remoteAuthority: getRemoteAuthority(uri)
+				gotoLineMode: true
```

Comment:
Intentional?

Reply:
I'm bringing it back.
I thought it was confusing/wrong that `getRemoteAuthority` is used on protocol URI, but then, getRemoteAuthority only returns something if the schema is `vscode-remote`.